src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php

```diff
@@ -130,7 +130,7 @@ private function decorateVar($var)
                     return new ClassStub($var);
                 }
             }
-            if (false !== strpos($var, DIRECTORY_SEPARATOR) && file_exists($var)) {
+            if (false !== strpos($var, DIRECTORY_SEPARATOR) && @file_exists($var)) {
```

Comment:
Indeed, I think it totally makes sense 👍 

I've also updated the test case which could be completed later in order to cover the base collector `cloneVar` behavior, but let me know if you think it's not that much useful.